File: Math/Easy/1137/1137.py
class Solution:
    def tribonacci(self, n: int) -> int:
        if n < 2:
            return n
        i = 0
        j = 1
        k = 1
        l = 0
        for idx in range(n - 2):
            l = i + j + k
            i = j
            j = k
            k = l
        return k

File: Math/Easy/1137/1137.c
int tribonacci(int n){
    if(n < 2){
        return n;
    }
    int i = 0, j = 1, k = 1;
    int l = 0;
    for(int idx = 0; idx < n-2; idx++){
        l = i + j + k;
        i = j;
        j = k;
        k = l;
    }
    return k;
}